docker/npm-build-pod/Dockerfile

```diff
@@ -0,0 +1,14 @@
+#invoke npm in jenkinsfile: sh "scl enable rh-nodejs6 'npm run build'" 
+FROM openshift/jenkins-slave-nodejs-rhel7:latest
+USER root
+RUN yum remove -y rh-nodejs4; \
+    yum repolist > /dev/null && \
+    yum-config-manager --enable rhel-server-rhscl-7-rpms && \
```

Comment:
Unless there's a good reason to permanently enable these repos, I'd suggest enabling them at "yum install" time below. This is a bit cleaner IMHO + there's a bug in RHEL 7.4 that makes the `yum-config-manager` misbehave - see similar change made in [this PR](https://github.com/redhat-cop/containers-quickstarts/pull/43).